command/agent_info_test.go

```diff
@@ -16,49 +16,43 @@ func TestAgentInfoCommand_Implements(t *testing.T) {
 func TestAgentInfoCommand_Run(t *testing.T) {
 	ci.Parallel(t)
 	srv, _, url := testServer(t, false, nil)
-	defer srv.Shutdown()
+	defer stopTestAgent(srv)
 
 	ui := cli.NewMockUi()
 	cmd := &AgentInfoCommand{Meta: Meta{Ui: ui}}
 
 	code := cmd.Run([]string{"-address=" + url})
-	if code != 0 {
-		t.Fatalf("expected exit 0, got: %d", code)
-	}
+	must.Zero(t, code)
```

Comment:
I won't block because of this, but don't we want these explanatory messages on failures?

Reply:
not when the context is obvious - this is like the test output equivalent of 
```
i++ // increment i 
```